pkg/solana/txm/pendingtx.go

```diff
@@ -60,9 +68,9 @@ type pendingTxContext struct {
 	cancelBy map[string]context.CancelFunc
 	sigToID  map[solana.Signature]string
 
-	broadcastedTxs      map[string]pendingTx // transactions that require retry and bumping i.e broadcasted, processed
-	confirmedTxs        map[string]pendingTx // transactions that require monitoring for re-org
-	finalizedErroredTxs map[string]pendingTx // finalized and errored transactions held onto for status
+	broadcastedProcessedTxs map[string]pendingTx // broadcasted and processed transactions that may require retry and bumping
```

Comment:
Thanks for the change! Kind of bothered me I didn't do this with the initial change but also didn't want to open a separate PR for it